config/routes.rb

```diff
@@ -34,6 +28,12 @@
       end
     end
     resources :evaluator_submission_assignments, only: [:index, :update, :create]
+    resources :evaluation_forms do
```

Comment:
removed the index route
```suggestion
    resources :evaluation_forms, except: [:index] do
```